src/api.ts

```diff
@@ -111,6 +121,10 @@ export async function create(req: Request): Promise<Response> {
 		status: 'running',
 		source: req.source,
 		start_timestamp: new Date(),
+		is_final: !req.draft,
+
+		// Only set semver if provided on the request
+		...(!!req.semver && { semver: req.semver }),
```

Comment:
@thgreasi added the version on the request to the PR. This is mostly for UX, so the user can see the release version, even while the deploy is "running"